treeTopView.py

```diff
@@ -0,0 +1,86 @@
+from operator import itemgetter
+class Node:
+    def __init__(self, info):
+        self.info = info
+        self.left = None
+        self.right = None
+        self.level = None
+
+    def __str__(self):
+        return str(self.info)
+
+
+class BinarySearchTree:
+    def __init__(self):
+        self.root = None
+
+    def create(self, val):

```

Comment:
better renamed to `insert`, can you implement a recursive version?

Reply:
the setup for class was given by the question. However, I'll think over the recursive implementation.